tests/test_util.py

```diff
@@ -1,13 +1,314 @@
 import pytest
 import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import xarray as xr
+import pandas as pd
+from metpy.units import units
+import metpy.calc as mpcalc
 
-from geocat.viz.util import truncate_colormap
+import geocat.viz.util as gv
```

Comment:
Was there a reason for switching this up?  The convention I've typically seen for code being tested is the explicit "from x import y" for all tested functions and classes.  Generally at the bottom of the imports section.

Reply:
I did not realize that, I thought importing every single util function would take up a lot of space, so I imported the module instead. I see how that would make it more clear right away.